tests/core/recipes/emt_recipes/test_emt_phonons.py

```diff
@@ -71,3 +72,33 @@ def test_phonon_flow_v4(tmp_path, monkeypatch):
     assert output["results"]["force_constants"].shape == (8, 8, 3, 3)
     assert "mesh_properties" in output["results"]
     assert output["atoms"] != atoms
+
+
+def test_phonon_flow_fixed(tmp_path, monkeypatch):
+    monkeypatch.chdir(tmp_path)
+    atoms = molecule("H2", vacuum=20.0)
+    output = phonon_flow(atoms, run_relax=True, min_lengths=5.0)
+
+    atoms1 = molecule("H2", vacuum=20.0)
+    atoms2 = molecule("H2", vacuum=20.0)
+
+    atoms2.positions += [10, 10, 10]
+
+    full_system = atoms1 + atoms2
+    full_system.set_constraint(FixAtoms(indices=[0, 1]))
+
+    output_fixed = phonon_flow(full_system, run_relax=True, min_lengths=5.0)
+
+    # Should be very close but not exactly the same
+    assert output["results"]["mesh_properties"]["frequencies"] == pytest.approx(
+        output_fixed["results"]["mesh_properties"]["frequencies"], rel=0.0, abs=1e-5
+    )
```

Comment:
@tomdemeyere: Is this PR good to go from your end?

Also, could you please explain this test for me? Why would the frequencies be the same if two of the atoms are fixed?

Reply:
The two molecules are far away, if you fix one of them it should not affect the other.

In the last one we fix one atom in each molecule giving completely wrong results...

The PR is good to go, let me keep in mind the get_phonopy problem I will come back to it if I find a solution